terraform/warehouse/ngwaf.tf

```diff
@@ -0,0 +1,57 @@
+provider "sigsci" {
+  # if i dont add this it errors: Provider configuration not present
+  alias      = "firewall"
+  corp       = "python"
+  email      = var.ngwaf_email
+  auth_token = var.ngwaf_token
+}
+
+resource "fastly_service_dictionary_items" "edge_security_dictionary_items" {
+  count         = var.activate_ngwaf_service ? 1 : 0
+  service_id    = fastly_service_vcl.pypi.id
+  dictionary_id = one([for d in fastly_service_vcl.pypi.dictionary : d.dictionary_id if d.name == var.edge_security_dictionary])
+  items = {
+    Enabled : "100"
```

Comment:
Can this be set to 50% for test, and 1% for prod?

Reply:
[eac3c1f](https://github.com/pypi/infra/pull/172/commits/eac3c1fb218d80306320305cd70aa93d4cc1e4b7)